Readers/FastqReader.cs

```diff
@@ -1,6 +1,7 @@
 using System;
```

Comment:
i haven't profiled this vs. the line reader yet. need to do that and retire one of them. the line reader was an experiment (and the code is cleaner / safer)

Reply:
👍🏻